### PR TITLE
[ISSUE #446] Use strong reference avoid weak references recycled when GC occurs

### LINKED ISSUE #446: Methods that continuously call WeakReference wrapper may throw an NPE
All methods using `WeakReference` must check whether it is `NULL` after calling `get()`.

image

### PR DESCRIPTION
**Pull request by zzzk1**
Fixes #446 

### Motivation
*Use strong references to replace the direct use of weak references to avoid NPE problems caused by weak references being restored during system GC occurs*

### Modifications

*Describe the modifications you've done.*

### Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by zzzk1** [comment]
@Pil0tXia pls take a look

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/group/ClientSessionGroupMapping.java [hunk 1]
```diff
@@ -260,8 +260,7 @@ private void startClientGroupConsumer(Session session) throws Exception {
         }
         synchronized (lockMap.get(subsystem)) {
             log.info("readySession session[{}]", session);
-            ClientGroupWrapper cgw = session.getClientGroupWrapper().get();
-
+            ClientGroupWrapper cgw = this.getClientGroupMap().get(session.getClient().getGroup());
             boolean flag = cgw != null && cgw.addGroupConsumerSession(session);
```
Comment (Pil0tXia) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/group/ClientSessionGroupMapping.java:264]: The key of `clientGroupMap` is `subsystem`, I'm not sure whether the `group` key here works well.

**Review by Pil0tXia — COMMENTED** [review]
Please be noted that `getClientGroupWrapper` has some other usages in EventMesh~

**Comment by Pil0tXia** [comment]
May I inquire if you have any questions?

**Comment by zzzk1** [comment]
> May I inquire if you have any questions?

Seems like it was not a problem https://github.com/apache/eventmesh/issues/446#issuecomment-882404842

**Comment by Pil0tXia** [comment]
@zzzk1 Thanks, you are welcome to pick other issue contributions!

**Pull request closed without merging**